spec/acceptance/basic_spec.rb

```diff
@@ -117,3 +117,41 @@ class { 'pulpcore':
   end
 
 end
+
+describe 'with content cache enabled' do
```

Comment:
Should this also verify Redis is running? Perhaps even perform some content request and verify if it's requested twice that a cache hit is returned (which I think should be in the response header)

Reply:
I thought about the checking the content cache itself, and this would be a good enhancement to our testing but I'd like to do that as a follow up as it may take more time to figure out getting all the right API calls lined up. I've added the Redis check.

Reply:
I also figured a real in depth check would be complex. It does mean adding actual content into our instance which we don't do today. Perhaps we can leverage pulp-cli for this when we do get to it.